sdk/go/common/resource/plugin/host.go

```diff
@@ -73,8 +73,6 @@ type Host interface {
 	// EnsurePlugins ensures all plugins in the given array are loaded and ready to use.  If any plugins are missing,
 	// and/or there are errors loading one or more plugins, a non-nil error is returned.
 	EnsurePlugins(plugins []workspace.PluginInfo, kinds Flags) error
-	// GetRequiredPlugins lists a full set of plugins that will be required by the given program.
-	GetRequiredPlugins(info ProgInfo, kinds Flags) ([]workspace.PluginInfo, error)
```

Comment:
Do you have any ideas on how we could integrate automatic plugin acquisition with the new world? Today, it's annoying but I think most users will be able to bear. As multi-lang and component consumption become more common, hand managing dependencies in this execution mode becomes problematic.

Reply:
In principle all we need to do is implement `GetRequiredPlugins` in each SDK's language service. Ideally we'd factor out the code that's currently used to do this, but it's not always in the SDK's target language (e.g. the implementation of the Node language provider is currently a Go binary that launches Node in response to `Run`).